tests/tests_unit/graph/test_loaders/test_asset_loader.py

```diff
@@ -0,0 +1,31 @@
+from rdflib import URIRef
+
+from cognite.neat.graph.examples import nordic44_knowledge_graph
+from cognite.neat.graph.extractors import RdfFileExtractor
+from cognite.neat.graph.issues.loader import InvalidInstanceError
+from cognite.neat.graph.loaders import AssetLoader
+from cognite.neat.graph.stores import NeatGraphStore
+from cognite.neat.rules.importers import ExcelImporter
+from tests import config
+
+
+def test_generation_of_assets():
+    store = NeatGraphStore.from_memory_store()
+    store.write(RdfFileExtractor(nordic44_knowledge_graph, base_uri=URIRef("http://purl.org/nordic44#")))
+
+    asset_rules, _ = ExcelImporter(filepath=config.DATA_FOLDER / "asset-architect-test.xlsx").to_rules()
```

Comment:
Based on the most recent failure of the coverage tests failing: https://github.com/cognitedata/neat/actions/runs/9971226811/job/27551846145?pr=542

It looks like the `asset_architect_test.xlsx` is not providing valid results thus making asset_rules None. Suggest to make this
```suggestion
    asset_rules = ExcelImporter(filepath=config.DATA_FOLDER / "asset-architect-test.xlsx").to_rules(errors="raise")
```

Then you should get better error messages at least.